libraries/WiFiProv/src/WiFiProv.cpp

```diff
@@ -26,6 +26,7 @@
 #include <esp_wifi.h>
 #include <esp_event.h>
 #include <esp32-hal.h>
+#include "qrcode.h"
```

Comment:
you should check if that include exists and only then include and provide the new function. When building as component and qrcode component is not additionally installed, compilation will fail: https://github.com/espressif/arduino-esp32/actions/runs/4989420314/jobs/8933324447?pr=8209#step:4:1408

Reply:
I was expecting this to be available for all SoCs, I can see that is not, but do not understand why?
How can I test it?

Reply:
it exists from the lib-builder, because it's installed with RainMaker, but as clean IDF+Arduino-as-component it does not exist for ANY chip.

Reply:
Should be resolved - CI passed